js/_website/generate_jsons/src/docs/__init__.py

```diff
@@ -97,9 +97,19 @@ def add_guides():
 add_guides()
 
 
+def convert_inline_code_syntax(raw_string: str) -> str:
+    return (raw_string
+            .replace(
+                "{",
+                "<span class='text-orange-500' style='font-family: monospace; font-size: large;' >",
+            )
+            .replace("}", "</span>"))
+
+
 def style_types():
     for mode in docs:
         for cls in docs[mode]:
+            cls["styled_description"] = convert_inline_code_syntax(cls["description"])
```

Comment:
To avoid overwriting the existing `description` field, I added a new field `styled_description` for this purpose.
This is different from the `tags.*` field because they are not overwritten as follows.
```json
{
  "description": "<raw description text>",
  "styled_description": "<styled description text>",  // Added in this PR, because the original `description` field is at the top level.
  "tags": {
    "preprocessing": "<raw preprocessing text>"
  },
  "preprocessing": "<styled preprocessing text>"  // This field added by this function, avoiding overwriting the original `tags.preprocessing` field.
}
```